playwright/pages/components/UIDEditor.ts

```diff
@@ -29,11 +29,13 @@ export class UIDEditor {
   /**
    * Dynamic locators
    */
-
   getDialogTrigger(label: string): Locator {
     return this.page.getByRole("button", { name: label, exact: true });
   }
 
+  getErrorMessage(message: string): Locator {
+    return this.dialog.getByText(message);
```

Comment:
💡 #idea

`exact` could be used here, right?

In best practice, we anyway said:
> Use `false` when you deliberately don't want an exact matching.

So it clearly shows if you thought about the best practice or not.

Reply:
fixed